composer.json

```diff
@@ -7,8 +7,8 @@
     "license": "Apache-2.0",
     "require": {
         "php": "^7.4|^8.0",
-        "google/auth": "^1.33",
-        "google/apiclient-services": "~0.200",
+        "google/auth": "^1.37",
+        "google/apiclient-services": "dev-universe-domain as 0.340",
```

Comment:
This needs to be updated to `^0.340` (or whatever version we end up tagging for the new universe domain support)